app/services/csv_imports/gias_import.rb

```diff
@@ -46,6 +46,7 @@ def call
         end
         rows += 1
       end
+      Rails.logger = Logger.new($stdout)
```

Comment:
Not sure if this is a good idea to override the `Rails.logger` in the service. Wouldn't it be better if the service defined it's own instance eg `@logger = Logger.new($stdout)` in the initialiser and then use that to log?

Reply:
Will give it a go